<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>312</width>
                <height>294</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>注册</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="text">
                                        <string>用户名：</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="register_username"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="text">
                                        <string>密码：</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="register_password"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label_3">
                                    <property name="text">
                                        <string>确认密码：</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="confirm"/>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <widget class="QPushButton" name="button_determine">
                            <property name="text">
                                <string>确定</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
